libs/sdk-core/src/swap_in/swap.rs

```diff
@@ -493,6 +502,21 @@ impl BTCReceiveSwap {
             .await?;
         get_utxos(address, transactions)
     }
+
+    /// Registers for a swap tx notification. When a new transaction to the specified `swap_address`
+    /// is confirmed, a callback will be triggered to the specified `webhook_url`.
+    pub(crate) async fn register_swap_tx_notification(
```

Comment:
This still has create visibility. Perhaps we don't need this function at all and can invoke the super::`register_swap_tx_notification` directly from the `create_swap_address` function?